### PR TITLE
Replace Magento with Adobe Commerce

### PR DESCRIPTION
**Pull request by BarnyShergold**
## Purpose of this pull request
 
This pull request (PR) fixes issue #57 & #56
 
## Affected Support KB pages
 
https://support.magento.com/hc/en-us/articles/360049171211-Best-practices-for-render-blocking-resources-in-Magento-
https://support.magento.com/hc/en-us/articles/4402349423757-Error-purging-cache-in-Magento-Commerce-Admin

Query - The products are now known as Adobe Commerce & Adobe Commerce Cloud - using just commerce and then referring to "our cloud infrastructure" might be confusing and it a little long winded?

### CODE REVIEW

File: src/best-practices/general/best-practices-for-render-blocking-resources-in-magento.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
 
-This article provides guidance on preventing resources blocking page rendering in Magento, which can lead to a significant increase in page rendering time and cause performance degradation.
+This article provides guidance on preventing resources blocking page rendering in Adobe Commerce, which can lead to a significant increase in page rendering time and cause performance degradation.
 
 ## Affected products and versions
 
-* Magento Commerce, all [supported versions](https://magento.com/sites/default/files/magento-software-lifecycle-policy.pdf)  
-* Magento Commerce Cloud, all [supported versions](https://magento.com/sites/default/files/magento-software-lifecycle-policy.pdf) 
+* Adobe Commerce on-premise, all [supported versions](https://magento.com/sites/default/files/magento-software-lifecycle-policy.pdf)  
+* Adobe Commerce Cloud, all [supported versions](https://magento.com/sites/default/files/magento-software-lifecycle-policy.pdf)
```
Comment (almarchenko) [inline src/best-practices/general/best-practices-for-render-blocking-resources-in-magento.md:11]:
```suggestion
* Adobe Commerce on our cloud infrastructure, all [supported versions](https://magento.com/sites/default/files/magento-software-lifecycle-policy.pdf) 
```

File: src/best-practices/general/best-practices-for-render-blocking-resources-in-magento.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
+If assistance is required or if there are questions or concerns, [submit a support ticket](https://support.magento.com/hc/en-us/articles/360019088251-Submit-a-support-ticket) .
 
 ## Related reading
 
-* [Magento User Guide > Optimizing Resource Files](https://docs.magento.com/user-guide/system/file-optimization.html)
-* KB [Optimize CSS and JS files Magento Commerce Cloud and Magento Commerce](https://support.magento.com/hc/en-us/articles/360044482152-CSS-and-Javascript-file-optimization-on-Magento-Commerce-Cloud-and-Magento-Commerce) 
-* DevDocs [Magento > Frontend Developer Guide > Cascading style sheets (CSS) > CSS merging, minification and performance](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css-overview.html#css-merging-minification-and-performance) 
+* [Adobe Commerce User Guide > Optimizing Resource Files](https://docs.magento.com/user-guide/system/file-optimization.html)
+* KB [Optimize CSS and JS files Adobe Commerce Cloud and Adobe Commerce](https://support.magento.com/hc/en-us/articles/360044482152-CSS-and-Javascript-file-optimization-on-Magento-Commerce-Cloud-and-Magento-Commerce)
```
Comment (almarchenko) [inline src/best-practices/general/best-practices-for-render-blocking-resources-in-magento.md:22]:
```suggestion
* KB [Optimize CSS and JS files Adobe Commerce](https://support.magento.com/hc/en-us/articles/360044482152-CSS-and-Javascript-file-optimization-on-Magento-Commerce-Cloud-and-Magento-Commerce) 
```

File: src/troubleshooting/general/error-purging-cache-in-admin.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
 ---
 
-This article explains how to identify the cause of an error message that occurs when purging the cache in Magento Commerce Admin. When you attempt to purge cache through the Admin you receive the following message:
+This article explains how to identify the cause of an error message that occurs when purging the cache in Adobe Commerce Admin. When you attempt to purge cache through the Admin you receive the following message:
 */app/project-id/pub/media/catalog/product/cache/directory/filename" file can't be deleted. Warning!unlink(/app/project id/pub/media/catalog/product/cache/directory/filename): No such file or directory*
 
 ## Affected products and versions
 
-Magento Commerce and Magento Commerce Cloud: 2.3.0-2.3.7, 2.4.0-2.4.2-p1
+Adobe Commerce and Adobe Commerce Cloud: 2.3.0-2.3.7, 2.4.0-2.4.2-p1
```
Comment (almarchenko) [inline src/troubleshooting/general/error-purging-cache-in-admin.md:11]:
```suggestion
Adobe Commerce, all deployment types: 2.3.0-2.3.7, 2.4.0-2.4.2-p1
```

**Review by almarchenko — CHANGES_REQUESTED** [review]

**Review by almarchenko — APPROVED** [review]

**Comment by almarchenko** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by magento-engcom-team** [comment]
@almarchenko the branch with code successfully imported into`magento-kb/knowledge-base` repository. Branch name: `imported-magento-knowledge-base-79`.

**Pull request merged**